### DISCUSSION TITLE
No sound playing

### DISCUSSION THREAD

**Discussion by rilysh** [Androz2091/discord-player#613] [Q&A]
After last update of Discord-Player everything is fine until today, I just found whenever I do any query it's says No Music Found on YouTube for this query. It's looks like ytdl-core issue I found a issue on their https://github.com/fent/node-ytdl-core/issues/962. 

It's not only YouTube video but also Spotify links too. It's playing nothing and throwing no errors, warnings on console and I'm not sure about it.

**Comment by twlite** [comment]
Spotify does not allow us to download songs like youtube because of which, spotify songs are indirectly fetched from youtube.

**Comment by VanshKumar2821** [comment]
I am Also having This Issue Of No Sound 🙄

**Comment by rilysh** [comment]
I'm still not sure is it was tempban from YouTube (but I doesn't make too many request for tracks), but now after almost 7 hours later it's fixed now. But I'm not sure if I again got it again in future...

**Comment by VanshKumar2821** [comment]
I Tried It On A New Bot With A New Project It Joined Voice Channel But Wasn't Playing Any Music When I Run The Command.

**Comment by julianvink17** [comment]
I'm having the exact same error, don't know how to fix it..

**Comment by twlite** [comment]
discord-player version?

**Comment by securing-ab** [comment]
Then try to install [this](https://www.npmjs.com/package/ytdl-core-discord)
